README.md

```diff
@@ -100,14 +100,40 @@ or
   git push -u origin main
 ```
 
-If you get an error message like the one below, you probably forgot to fork the repository before cloning it. It is best to start over and fork the project repository first.
+If you get an error message like the one below, you probably forgot to fork the repository before cloning it.
 
 ```bash
 ERROR: Permission to EddieHubCommunity/hacktoberfest-practice.git denied to <your-github-username>.
 fatal: Could not read from remote repository.
 Please make sure you have the correct access rights and the repository exists.
 ```
 
+No need to panic or loose your streak at this point. Follow along to fix this issue.
+
+To fix this issue, fork the project using the gray [`Fork`](https://github.com/EddieHubCommunity/hacktoberfest-practice/fork) button in the top right of this page.
+Once the fork is complete, it will redirect you to the newly forked repository on your GitHub account.
+Run these commands at this point, and you are good to continue:
+
+```
```

Comment:
Should we add `bash` after the ` ``` `? That way, it may have formatting on the commands we use.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Not everyone uses Bash. I think this should be left generic for the developer to figure out and its not whats the scope of this README section.

Reply:
> Not everyone uses Bash. I think this should be left generic for the developer to figure out and its not whats the scope of this README section.

That cannot be seen by users unless they check the code directly.
It is used to determine what language you're using in the codeblock. We can use `shell` if you want.